src/main/java/com/rackspace/ceres/app/web/WriteController.java

```diff
@@ -21,7 +21,13 @@
 import com.rackspace.ceres.app.model.PutResponse;
 import com.rackspace.ceres.app.model.TagFilter;
 import com.rackspace.ceres.app.services.DataWriteService;
+
```

Comment:
You can remove this file from the diff?

Reply:
By mistake, I had committed my changes with unused imports. I'll make sure that this is not repeated.